commands/Skill/bugjuice.js

```diff
@@ -0,0 +1,14 @@
+module.exports.execute = async (client, message) => {
+  return await message.channel.send(
+    Math.random() < 0.5 ? ':cockroach:' : ':cocktail:'
+  );
+};
+
+module.exports.config = {
+  name: 'bug juice',
+  aliases: [],
+  module: 'Fun',
+  description:
+    'I will choose either :cockroach: or :cocktail:. (My version of Russian Roulette)',
+  usage: ['coffee'],
```

Comment:
Update this to be the usage of your command. Since you don't use any args, just leave it as the name of your command.